standalone/pruntime/enclave/src/system/mod.rs

```diff
@@ -352,12 +352,16 @@ impl System {
         let sender = MessageOrigin::Worker(pubkey.clone());
         // TODO: create gk_egress dynamically with gk masterkey
         let gk_egress = send_mq.channel(MessageOrigin::Gatekeeper, pair.clone());
+
```

Comment:
See the TODO above, I think it's time to make the mq egress channel with the masterkey now.

Reply:
Then we need to verify the signature here:
https://github.com/Phala-Network/phala-blockchain/blob/119d2ea9a0b12b6d4b0298358b17cbfc665cbfd3/pallets/phala/src/registry.rs#L297